test/core/reporters/v1.js

```diff
@@ -283,4 +283,25 @@ describe('reporters - v1', function() {
 			done();
 		});
 	});
+	it('should add version information', function(done) {
+		axe.run(optionsV1, function(err, results) {
+			try {
+				assert.isNull(err);
+				assert.isObject(results.testEngine, 'tesetEngine');
+				assert.equal(results.testEngine.name, 'axe-core');
```

Comment:
Why are we testing the same thing 3 different times? Is there a case where 1 of these tests _could_ fail but the others wouldn't? If not, I think we only need to do this once.

Reply:
These are different reporters. Just answered my own question. Disregard my question.